Intersect.Client/MonoGame/Audio/MonoMusicSource.cs

```diff
@@ -33,11 +34,25 @@ public class MonoMusicSource : GameAudioSource
 
         
 
-        public MonoMusicSource(string path, string realPath)
+        public MonoMusicSource(string path, string realPath, string name = "")
```

Comment:
`string.Empty` or if that doesn't work `default`

then `string.IsEmptyOrWhiteSpace(name) ? string.Empty : name`